config/main.py

```diff
@@ -3369,37 +3369,67 @@ def get_acl_bound_ports():
 
     return list(ports)
 
-#
-# 'table' subcommand ('config acl add table ...')
-#
 
-@add.command()
-@click.argument("table_name", metavar="<table_name>")
-@click.argument("table_type", metavar="<table_type>")
-@click.option("-d", "--description")
-@click.option("-p", "--ports")
-@click.option("-s", "--stage", type=click.Choice(["ingress", "egress"]), default="ingress")
-def table(table_name, table_type, description, ports, stage):
+def expand_vlan_ports(port_name):
     """
-    Add ACL table
+    Expands a given VLAN interface into its member ports.
+
+    If the provided interface is a VLAN, then this method will return its member ports.
+
+    If the provided interface is not a VLAN, then this method will return a list with only
+    the provided interface in it.
     """
     config_db = ConfigDBConnector()
     config_db.connect()
 
+    vlan_members = config_db.get_keys("VLAN_MEMBER")
```

Comment:
>vlan_members [](start = 4, length = 12)

Did you test an empty vlan? an non-existing vlan?

Reply:
Added.